quinn-proto/src/crypto/rustls.rs

```diff
@@ -260,13 +263,14 @@ impl crypto::ClientConfig for rustls::ClientConfig {
                 rustls::quic::ClientConnection::new(
                     self,
                     version,
-                    server_name
-                        .try_into()
-                        .map_err(|_| ConnectError::InvalidDnsName(server_name.into()))?,
+                    ServerName::try_from(server_name)
+                        .map_err(|_| ConnectError::InvalidDnsName(server_name.into()))?
```

Comment:
Should we rename this error variant now that `server_name` can be an IP?

Reply:
Done.